CMakeLists.txt

```diff
@@ -164,7 +167,7 @@ if(HIP_FOUND AND Libva_FOUND)
   # make test with CTest
   enable_testing()
   include(CTest)
-  add_subdirectory(samples)
+  add_subdirectory(test)
```

Comment:
@kiritigowda  There is a Cmake error with this change. have you tested this on your local machine?

Reply:
@AryanSalmanpour -- see logs below. Are you seeing any issues?

Reply:
@kiritigowda I see this error with this PR. The develop branch ToT works fine.

cd rocDecode
mkdir build;cd build

cmake ..
-- The CXX compiler identification is unknown
CMake Error at CMakeLists.txt:33 (project):
  The CMAKE_CXX_COMPILER:

    clang++

  is not a full path and was not found in the PATH.

  Tell CMake where to find the compiler by setting either the environment
  variable "CXX" or the CMake cache entry CMAKE_CXX_COMPILER to the full path
  to the compiler, or to the compiler name if it is in the PATH.


-- Configuring incomplete, errors occurred!

Reply:
Is your OS Ubuntu 22.04?

Reply:
@AryanSalmanpour - CI results - http://math-ci.amd.com/blue/organizations/jenkins/mainline%2Fprecheckin%2FrocDecode/detail/PR-408/2/pipeline/549/


Need to reproduce your issue as I see the AzureCI failing with the same issue

Reply:
Yes, my OS is UB 22.04 + mainline Build#14571